crawl-ref/source/dat/des/branches/spider.des

```diff
@@ -2091,6 +2118,157 @@ xx...x...xx
     xxx
 ENDMAP
 
+NAME:   nicolae_spider_broodchamber
+TAGS:   transparent patrolling
+DEPTH:  Spider
+ORIENT: south
+KITEM:  M = * / | w:3
+KITEM:  PD = % / $ w:5 / * w:5 / nothing w:20
+KITEM:  Q = % / nothing
+KMONS:  M = broodmother
+KMONS:  P = pharaoh ant
+KMONS:  D = culicivora
+KMONS:  Q = entropy weaver / torpor snail / moth of wrath / ghost moth / \
+            orb spider / spark wasp / jumping spider / jorogumo / sun moth / \
+            boulder beetle
+KFEAT:  MPDQ! = web / . w:50
+KMASK:  MPDQ! = no_monster_gen
+SUBST:  ` = xx!, - = xxs
+NSUBST: S = 1=M / 2=M! / !, s = 2=P / 2=D / 1=Q / 2=PDQ! / !
+MAP
+    xxxxxxxx@@xx
+   xx```````..`x
+   x`!!.......`x
+   x`!!.......`x
+   x`!!```````xx
+  xxx!!xxxxxxxx
+ xx-!!!!-xx
+xx-ssssss-xx
+x-ssssssss-x
+x-ssssssss-x
+x-ssssssss-x
+x-ssssssss-x
+xx-sSSSSs-xx
+ xx-SSSS-xx
+  xxxxxxxx
+ENDMAP
+
+NAME:   nicolae_spider_brood_tunnel
+TAGS:   transparent patrolling
+DEPTH:  Spider
+ORIENT: float
+KMONS:  M = broodmother band
+SUBST:  - = x., ` = x..., % = %:60 .:30 *:20 |, S = 009...
+MAP
+ xxxxx  xxxxxx   xxxxx
+xx---xxxxx---xxxxx---xx
+x--%--xxx--%--xxx--%--x
+x-%M%--x--%M%--x--%M%-x
+x--%`.---.`%`.---.`%--x
+xx--.`.-.`.-.`.-.`.--xx
+ xx--.`.`.---.`.`.--xx
+ xxx--S`.--x--.`S--xxx
+ xx--.`.--xxx--.`.--xx
+xx--.`.--xxxxx--.`.--xx
+x--.`.--xxxxxxx--.`.--x
+x-.`S--xxxxxxxxx--S`.-x
+x--.`.--xxxxxxx--.`.--x
+xx--.`.--xxxxx--.`.--xx
+ xx--.`.--xxx--.`.--xx
+ xxx--S`.-xxx-.`S--xxx
+ xx--.`.--xxx--.`.--xx
+xx--.`.--xxxxx--.`.--xx
+x--.`.--xxxxxxx--.`.--x
+x-.`S--xxxxxxxxx--S`.-x
+ @..--xxxxxxxxxxx--..@
+  @--xxxx     xxxx--@
+   xxxx         xxxx
+ENDMAP
+
+NAME:   nicolae_spider_obnoxious_hallway
+TAGS:   transparent
+DEPTH:  Spider
+ORIENT: float
+KMONS:  S = jorogumo / steelbarb worm / tarantella / \
+            jumping spider / torpor snail / nothing w:30
+KFEAT:  @ = W / .
+SUBST:  M = Wx, ' = x.
+MAP
+ xxxxxxxxxxxxxxxxxxxxxx
+ MMxx''xxxxxxxxxx''xxMM
+@WWMMSSxxxxxxxxxxSSMMWW@
+@WWWWWMxx''xx''xxMWWWWW@
+ MMWWWWMMSSxxSSMMWWWWMM
+ xxMWWWWWWMxxMWWWWWWMxx
+ xxSSMWWWWWMMWWWWMMSSxx
+ xx''xxMWWWWWWWWMxx''xx
+  xxxxxSSMMWWMMSSxxxxx
+   xxxx''xxMMxx''xxxx
+    xxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:   nicolae_spider_solar_radiation
```

Comment:
All of the spider monster vaults seemed pretty nasty but ultimately fair. I think this one is probably closest to being "too much" just because it's small; probably ok but you could drop the monster density a bit imo.